features/admin.claims.v1/components/claims-list.tsx

```diff
@@ -230,6 +231,9 @@ export const ClaimsList: FunctionComponent<ClaimsListPropsInterface> = (
     const hasAttributeUpdatePermissions: boolean = useRequiredScopes(featureConfig?.attributeDialects?.scopes?.update);
     const hasAttributeDeletePermissions: boolean = useRequiredScopes(featureConfig?.attributeDialects?.scopes?.delete);
 
+    const { isSubOrganization } = useGetCurrentOrganizationType();
+    const isReadOnly: boolean = isSubOrganization() || !hasAttributeUpdatePermissions;
```

Comment:
Don't we need to consider `hasAttributeCreatePermissions` too?

```suggestion
    const isReadOnly: boolean = isSubOrganization() || !hasAttributeUpdatePermissions || !hasAttributeCreatePermissions;
```

Reply:
Create permission is not related to this.